### ISSUE TITLE
Building testConfig fails to find uid when using custom integration folder

### ISSUE THREAD

**Issue by shiva** [prescottprue/cypress-firebase#42]
When using a custom integration folder, building testConfig fails as follows:

```
❯ yarn build:testConfig
$ cypress-firebase createTestEnvFile
✖ Error: Test env file could not be created:
STAGE_TEST_UID is missing from environment. Confirm that tests/e2e/integration/config.json contains either STAGE_TEST_UID or TEST_UID. 
```

cypress.json:
```javascript
{
  "projectId": "blah",
  "pluginsFile": "tests/e2e/plugins/index.js",
  "integrationFolder": "tests/e2e/integration/",
  "screenshotsFolder": "test/e2e/screenshots/",
  "videosFolder": "test/e2e/videos/",
  "supportFile": "test/e2e/support/index.js",
  "fixturesFolder": "test/e2e/fixtures/"
}
```

**Comment by shiva** [comment]
https://github.com/prescottprue/cypress-firebase/blob/e2cf6fe69f4dfe8abcc34fd359a544ced3138a8c/src/utils.js#L89 seems to be the issue. We check for hard-coded path, instead of resolving the base path using getCypressFolderPath().

**Comment by prescottprue** [comment]
Included in v0.2.2, thanks for the PR
